x/ecocredit/simulation/params.go

```diff
@@ -7,13 +7,14 @@ import (
 
 	simtypes "github.com/cosmos/cosmos-sdk/types/simulation"
 	"github.com/cosmos/cosmos-sdk/x/simulation"
+
 	"github.com/regen-network/regen-ledger/x/ecocredit"
 	"github.com/regen-network/regen-ledger/x/ecocredit/core"
 )
 
 // ParamChanges defines the parameters that can be modified by param change proposals
 // on the simulation
-func ParamChanges(r *rand.Rand) []simtypes.ParamChange {
+func ParamChanges(_ *rand.Rand) []simtypes.ParamChange {
```

Comment:
If this is unused, should we remove the parameter?

Reply:
https://github.com/regen-network/regen-ledger/pull/1043/commits/3fcdf55d2eaa24f71f9e3c2698850a38773d22f8